cmd/opm/alpha/bundle/build.go

```diff
@@ -23,8 +25,8 @@ func newBundleBuildCmd() *cobra.Command {
 		Use:   "build",
 		Short: "Build operator bundle image",
 		Long: `The "opm alpha bundle build" command will generate operator
-        bundle metadata if needed and build bundle image with operator manifest
-        and metadata for a specific version.
+        bundle metadata and create/validate CSV if absence and build bundle image 
```

Comment:
I really like the idea of linking a doc, but consider our doc site is not up and the only available document is in a PR against repo's ReadMe, i wonder if this can be done after the doc site is done. For now, i hope the updated decription is suffcieint.

Reply:
Punting on a doc link is fine -- let's just correct the current grammar: `absence` -> `absent`, `build bundle` -> `build a bundle`